lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2013/ashrae_90_1_2013.Space.rb

```diff
@@ -160,7 +160,7 @@ def space_daylighting_fractions_and_windows(space,
           sensor_1_window = sorted_skylights[0]
         else
           # Sensor 1 controls toplighted area
-          sensor_1_frac = areas['toplighted_area'] / space_area_m2
+          sensor_1_frac = areas['toplighted_area'] / space_area_m2 / 2
```

Comment:
Correct an issue that made this case control pretty much 100% of lights since the same fraction was applied to both sensors.

Reply:
I'm a little confused at why the /2 is there.  
Something like:
```
num_sensors = 2
sensor_1_frac = areas['toplighted_area'] / space_area_m2 / num_sensors
```
would make more sense